tests/queries/0_stateless/03640_skip_indexes_with_or.sql

```diff
@@ -0,0 +1,105 @@
+-- Tests for skip index with predicate conditions containing AND and OR.
+
+SET parallel_replicas_local_plan = 1; -- test verifies EXPLAIN output
+
+DROP TABLE IF EXISTS tab1;
```

Comment:
(minor: let's use just `tab` instead of `tab1` and `tab2`. You can drop `tab` then in l. 79 and create a second table of the name name)